### DISCUSSION TITLE
Getting back a github account

### DISCUSSION THREAD

**Discussion by nabil-el-allaouii** [community/community#146616] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone , i hope youre all doing well , i have a question concerning a github account that i had , so basically someone got into my gmail and reseted the password of my account and also removed the linked email in it , and now i dont have any access to that account, is it possible to get that account back or there is no solution for this issue?

**Comment by MindfulLearner** [comment]
In these kind of cases you should contact Github by proving your identity and then you can get back everything.

**Reply by nabil-el-allaouii** [reply]
Thanks for your reply, i tried to add a ticket explaining the issue i have , but i still didnt get any answer yet.

**Reply by MindfulLearner** [reply, marked as answer]
Technically, there's no other option since is an issue based on getting hacked** 
And then you have to prove that youre the owner.

anyway maybe these could help you. 

useful websites:
https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy
https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

Ticket: 
https://support.github.com/contact/account

i hope that might help you !

good luck!

**Reply by nabil-el-allaouii** [reply]
thank you i will try what you told me , much appreciated .

**Reply by MindfulLearner** [reply]
ty! if that helped you can you give me answear in case! 

thank you and good luck
